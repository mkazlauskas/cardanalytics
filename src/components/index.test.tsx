import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Component from '.';

describe('Root component', () => {
  it('should match snapshot', () => {
    expect(toJson(shallow(<Component />))).toMatchSnapshot();
  });
});