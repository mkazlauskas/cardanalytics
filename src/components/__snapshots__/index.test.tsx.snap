// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root component should match snapshot 1`] = `
<WithStyles(ForwardRef(Grid))
  className="makeStyles-container-1"
  container={true}
>
  <WithStyles(ForwardRef(Card))
    className="makeStyles-card-2"
    key="General Cardano Info"
  >
    <WithStyles(ForwardRef(CardHeader))
      title="General Cardano Info"
    />
    <WithStyles(ForwardRef(CardContent))>
      <GeneralInfo />
    </WithStyles(ForwardRef(CardContent))>
  </WithStyles(ForwardRef(Card))>
  <WithStyles(ForwardRef(Card))
    className="makeStyles-card-2"
    key="Current Fees"
  >
    <WithStyles(ForwardRef(CardHeader))
      title="Current Fees"
    />
    <WithStyles(ForwardRef(CardContent))>
      <CurrentFees />
    </WithStyles(ForwardRef(CardContent))>
  </WithStyles(ForwardRef(Card))>
</WithStyles(ForwardRef(Grid))>
`;
